tensorflow/core/common_runtime/executor.cc

```diff
@@ -258,8 +262,8 @@ struct NodeItem {
     return reinterpret_cast<EdgeInfo*>(var());
   }
   AllocatorAttributes* output_attr_base() const {
-    return reinterpret_cast<AllocatorAttributes*>(var() + sizeof(EdgeInfo) *
-                                                              num_output_edges);
+    return reinterpret_cast<AllocatorAttributes*>(
+        var() + sizeof(EdgeInfo) * num_output_edges);
```

Comment:
Another whitespace-only change, please revert.

Reply:
Oh my bad, for some reason, clang-format did this change. I'll revert

Reply:
Fixed in 8b5edac86cb05a41d2de727dbb9e9f0f2d08c3db